modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -255,4 +239,54 @@ protected function getSqlStatement(string $file_path, string $table_name, array
     ]);
   }
 
+  /**
+   * Detect the CSV delimiter.
+   *
+   * @param string $file_path
+   *   Path to CSV file to analize.
+   *
+   * @return string
+   *   The delimiter to use for parsing file.
+   */
+  protected function detectDelimiter($file_path) {
```

Comment:
Looks like you merged the delimiter detector as well.

Reply:
Agh 🤦🏽

Reply:
@paul-m fixed